docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,8 @@ Version history
 
 1.9.0 (pending)
 ===============
+* Renamed the `RequestInfo` namespace to `StreamInfo` to better match its
```

Comment:
@venilnoronha nit: could you please prefix this item with sub system name as documented in https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md#release-notes?

Reply:
Addressed via https://github.com/envoyproxy/envoy/pull/4609/files#diff-d8d3e33358b55e6c0466dd1bb5f40c79R6 in #4609.

Reply:
Thank you :-)